public/app/plugins/panel/nodeGraph/layeredLayout.js

```diff
@@ -10,20 +10,14 @@ import {
 } from '@msagl/core';
 import { parseDot } from '@msagl/parser';
 
-addEventListener('message', async (event) => {
```

Comment:
moving this to separate file isn't needed for the fix itself but it is how the other layout.worker file is structured and makes it bit easier to test.